src/stores/commandStateStore.ts

```diff
@@ -11,6 +11,8 @@ const commandStateStore = reactMinistore<CommandState>({
   italic: false,
   underline: false,
   strikethrough: false,
+  foreColor: 'rgb(227, 227, 227)',
+  backColor: 'rgb(0, 0, 0)',
```

Comment:
We can't hardcode these colors in, as they depend on whether the Light or Dark theme is active.

You should probably be one of `false`, `null`, or `undefined` by default.